src/java.xml/share/classes/com/sun/org/apache/xerces/internal/impl/PropertyManager.java

```diff
@@ -250,15 +243,9 @@ public void setProperty(String property, Object value) {
             return;
         }
 
-        //check if the property is managed by security manager
-        if (fSecurityManager == null
-                || !fSecurityManager.setLimit(property, JdkProperty.State.APIPROPERTY, value)) {
-            //check if the property is managed by security property manager
-            if (fSecurityPropertyMgr == null
-                    || !fSecurityPropertyMgr.setValue(property, FeaturePropertyBase.State.APIPROPERTY, value)) {
-                //fall back to the existing property manager
-                supportedProps.put(property, value);
-            }
```

Comment:
`JdkProperty` and `FeaturePropertyBase` can be removed from `import`. Applies to other sources as well